src/main/java/se/sundsvall/supportmanagement/integration/db/AttachmentRepository.java

```diff
@@ -0,0 +1,12 @@
+package se.sundsvall.supportmanagement.integration.db;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import se.sundsvall.supportmanagement.integration.db.model.AttachmentEntity;
+
+import java.util.Optional;
+
+
+public interface AttachmentRepository extends JpaRepository<AttachmentEntity, String> {
+
+	Optional<AttachmentEntity> findById(String id);
```

Comment:
findById is already defined in JpaRepository. This method declaration is redundant.